CMakeLists.txt

```diff
@@ -13,10 +13,14 @@ set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
 add_definitions(-DIVAN_VERSION="${PROJECT_VERSION}" -DUSE_SDL)
 
 option(BUILD_MAC_APP "Build standalone application for MacOS" OFF)
+option(FORCE_HOME_AS_STATE_DIR "Statedir will be /.ivan/ in current user's homedir" OFF)
```

Comment:
`USE_HOME_FOR_STATE_DIR` is a better name.

Reply:
As you wish. I changed that. :sunglasses: 
Besides, I have reverted the changes to the **.travis.yml** file (I wasn't sure they were okay).

Reply:
I'm sorry, you were right. Fixed. :neutral_face: